index.html

```diff
@@ -74,6 +75,220 @@ <h1 class="main-text">I’m Giovanni Glad to see you!</h1>
             </div>
         </section>
     </div>
-   
+
+    <!-- Work Section -->
+    <section>
```

Comment:
- [ ] Kindly add all your project `section` cards into one `section` tag to follow HTML semantic tags due to SEO purpose 😊 like below 👍

```
<section>
  <section>Card 1</section>
  <section>Card 2</section>
  <section>Card 3</section>
  <section>Card 4</section>
</section>
```